nl/references/judges/index.md

```diff
@@ -6,7 +6,7 @@ order: 6
 
 # Judges
 
-Op deze pagina vind je een overzicht van alle judges die beschikbaar zijn op Dodona. Een judge is een stukje software dat verantwoordelijk is om de oplossingen van studenten te testen en de resultaten hiervan weg te schrijven in een formaat dat Dodona begrijpt.
```

Comment:
In this context, _this page_ refers to the current documentation page. While it might be useful to add a link to the page on Dodona, it should probably be done as an additional link.